introduction.md

```diff
@@ -203,7 +203,7 @@ curl -X POST -H <span class="str">"X-API-Token: yourToken"</span> http://rudder.
 
 Instead of passing parameters one by one, you can instead supply a JSON object containing all you want to do. You'll also have to set the *Content-Type* header to **application/json** (without it the JSON content would be ignored).
 
-The supplied file should be valid JSON: strings need quotes, booleans and integers don't, ...
+The supplied file must contains a valid JSON object: strings need quotes, booleans and integers don't, ...
```

Comment:
contain

Reply:
Must be a valid JSON (no need to add object)